### PR TITLE
Removing the work around jolokia and it's config file(s)

### PR DESCRIPTION
**Pull request by jluhrsen**
the current method for bringing jolokia in to OpenDaylight is just to
install the feature "odl-jolokia". This is probably something to be done
on a use-case basis and not added to the default features list.

Signed-off-by: Jamo Luhrsen <jluhrsen@redhat.com>

### CODE REVIEW

**Comment by jluhrsen** [comment]
BTW, there is other "clusering" stuff in puppet-opendaylight that might not be
needed. the recommended way to configure clustering is with the config scripts
shipped with the ODL distro. those scripts do all the work with the akka, etc 
files. but, if the consumers of puppet-opendaylight are not using those config
scripts at this point, removing other things here could break them.

also, it should be noted that removing these jolokia steps/files/etc will not 
break clustering. it's an additional feature needed to turn on some things 
that external tools (like our cluster monitor, or CSIT) use. but, as far as HA
working, it will still be fine if jolokia is not installed.

File: spec/spec_helper.rb [hunk 1]
```diff
@@ -206,21 +206,8 @@ def enable_ha_tests(options = {})
...
-          'path'  => '/opt/opendaylight/deploy/jolokia.xml',
-          'owner' => 'odl',
-          'group' => 'odl'
-        )
-      }
-    else
-      # Check for HA_NODE_COUNT < 2
-      fail("Number of HA nodes less than 2: #{ha_node_count} and HA Enabled")
-    end
+    # Check for HA_NODE_COUNT < 2
```
Comment (dfarrell07) [inline spec/spec_helper.rb]: I think you're missing a conditional here. This results in "if enable_ha flag is set, fail", which is causing the Travis CI build errors you see on the PR.

File: spec/spec_helper_acceptance.rb [hunk 2]
```diff
@@ -289,18 +289,8 @@ def enable_ha_validations(options = {})
...
-      it { should be_file }
-      it { should be_owned_by 'odl' }
-      it { should be_grouped_into 'odl' }
-    end
-    else
-      # Check for HA_NODE_COUNT < 2
-      fail("Number of HA nodes less than 2: #{ha_node_count} and HA Enabled")
-    end
+    # Check for HA_NODE_COUNT < 2
+    fail("Number of HA nodes less than 2: #{ha_node_count} and HA Enabled")
```
Comment (dfarrell07) [inline spec/spec_helper_acceptance.rb]: Same here, missing conditional. We're not actually calling this helper in the beaker test it turns out, so it's not causing a failure (`bundle exec rake cent_6test_dock` passes).

**Comment by dfarrell07** [comment]
> the recommended way to configure clustering is with the config scripts shipped with the ODL distro

I guess we should consume these scripts in puppet-odl and ansible-odl to do clustering config.

File: spec/spec_helper_acceptance.rb (hunk 2 above)
Comment (jluhrsen) [inline spec/spec_helper_acceptance.rb]: not sure if I'm pushing these patches right. not used to the github pr workflow. but, I added the conditional now.

**Comment by dfarrell07** [comment]
Looks good, passing rspec and beaker. Thanks @jluhrsen!

**Pull request merged**